src/ima.js

```diff
@@ -521,16 +527,10 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
     this.logger.debug('reset');
     this.eventManager.removeAll();
     this._adBreaksEventManager.removeAll();
-    this._hideAdsContainer();
     if (!this._isImaSDKLibLoaded()) {
       return;
     }
-    if (this._adsManager) {
-      this._adsManager.destroy();
-    }
-    if (this._adsLoader && !this._contentComplete) {
-      this._adsLoader.contentComplete();
-    }
+    this._destroyIMAManagers();
```

Comment:
you are changing the order here, meaning we won;t hide the adsContainer in case `_isImaSDKLibLoaded` is false.
We had issues around this in the past.

Reply:
@OrenMe right, fixed.